test/gracenote/response_test.rb

```diff
@@ -97,15 +86,23 @@ module Gracenote
         <GN_ID>153857542-F5B1994A6C9EBF79D7FFB02EDF189A1D</GN_ID>
```

Comment:
__[rubocop]__
* Separate every 3 digits in the integer portion of a number with underscores(_). [:ref](https://github.com/bbatsov/ruby-style-guide#underscores-in-numerics)